<launch>

  <!-- args corresponding to environment variables -->
  <arg name="UAV_NAME" default="$(env UAV_NAME)" />

    <!-- other args -->
  <arg name="standalone" default="true" />

  <arg unless="$(arg standalone)" name="nodelet" value="load" />
  <arg     if="$(arg standalone)" name="nodelet" value="standalone" />
  <arg unless="$(arg standalone)" name="nodelet_manager" value="$(arg UAV_NAME)_vision_nodelet_manager" />
  <arg     if="$(arg standalone)" name="nodelet_manager" value="" />

  <group ns="$(arg UAV_NAME)">

    <node name="automatic_start_darpa" pkg="nodelet" type="nodelet" args="$(arg nodelet) mrs_general/AutomaticStartDarpa $(arg nodelet_manager)" output="screen">

      <rosparam file="$(find mrs_general)/config/automatic_start_darpa.yaml" />

      <param name="scripts_path" value="$(optenv HOME)" />

        <!-- Subscribers -->
      <remap from="~mavros_state_in" to="mavros/state" />
      <remap from="~control_manager_diagnostics" to="control_manager/diagnostics" />
      <remap from="~odometry_in" to="odometry/odom_main" />

        <!-- services -->
      <remap from="~takeoff_out" to="uav_manager/takeoff" />
      <remap from="~goto_out" to="forest_flier/goto" />
      <remap from="~change_estimator_out" to="odometry/change_estimator_type_string" />
      <remap from="~change_hdg_estimator_out" to="odometry/change_hdg_estimator_type_string" />
      <remap from="~shutdown_in" to="~shutdown" />
      <remap from="~land_out" to="uav_manager/land" />
      <remap from="~return_out" to="forest_flier/return" />

    </node>

  </group>

</launch>
